app/utilities/schema.py

```diff
@@ -139,15 +134,17 @@ def _load_schema_file(schema_name, language_code):
             language_code=language_code,
         )
 
-    if not schema_exists(language_code, schema_name):
+    if not _schema_exists(language_code, schema_name):
         logger.error(
             "no schema file exists",
             schema_name=schema_name,
             language_code=language_code,
         )
         raise FileNotFoundError
 
-    schema_path = get_schema_file_path(schema_name, language_code)
+    schema_path = get_schema_path_map(include_test_schemas=True)[language_code][
```

Comment:
To be consistent with the variable name used earlier:
```suggestion
    schema_path_map = get_schema_path_map(include_test_schemas=True)[language_code][
```

Reply:
Done. Updated another 2 also.

Reply:
Have corrected the incorrect variable names in some instances.